app/src/main/res/layout/bottom_sheet_event.xml

```diff
@@ -173,7 +173,8 @@
                     android:textColor="@color/accent_yellow"
                     android:textStyle="bold"
                     android:layout_marginStart="5dp"
-                    android:visibility="gone">
+                    android:visibility="gone"
+                    tool:visibility="gone">
```

Comment:
tools:visibility="visible" so developers know it's there

Reply:
Updated